doc/source/rllib/rllib-training.rst

```diff
@@ -1237,12 +1238,45 @@ do:
 The level of parallelism within the evaluation step is determined via the ``evaluation_num_workers``
 setting. Set this to larger values if you want the desired evaluation episodes or timesteps to
 run as much in parallel as possible. For example, if your ``evaluation_duration=10``,
-``evaluation_duration_unit=episodes``, and ``evaluation_num_workers=10``, each eval RolloutWorker
+``evaluation_duration_unit=episodes``, and ``evaluation_num_workers=10``, each RolloutWorker
 only has to run 1 episode in each evaluation step.
```

Comment:
Can you explain where 1 is coming from? (e.g. evaluation_duration / evaluation_num_workers)? what happens if they don't divide? is there any error or is there some expected behavior?

Reply:
done